src/main/java/FileContent.java

```diff
@@ -0,0 +1,41 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Class for reading and writing content from File
+ */
+public class FileContent implements Content {
+
+    private final File file;
+    private final String charsetName;
+
+    /**
+     * Constructor for initializing FileContent object
+     * @param file file for reading and writing content
+     * @param charsetName charset for reading and writing content from File
+     */
+    public FileContent(File file, String charsetName) {
+        this.file = file;
+        this.charsetName = charsetName;
+    }
+
+    /**
```

Comment:
@ArthurRomasyuk 
It is an override method, no need to provide doc here. It is better to mark it with `@Override` annotation, same for second method